### DISCUSSION TITLE
2007/04/28/icing-on-the-cake

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#155] [Announcements]
# 2007/04/28/icing-on-the-cake

这是今天在信里看到的一句成语。Icing 是蛋糕外面的那层糖衣，想起蛋糕已经可以流口水了，再加一层糖衣那就更别提多馋人了。所以这个短语翻译成中文就是“锦上添花”。

https://blog.qizhen.xyz/2007/04/28/icing-on-the-cake

**Comment by ruanqizhen** [comment]
之前博客下的留言：

xia
2007年04月29日 00:05 编辑
还是这样学英语比较记得住~哈~

回复
永青
2007年04月28日 09:06 编辑
心眼真坏啊，以后谁升职加薪还敢跟你说啊。。。哈哈

回复
